pkg/operator/testlib/volumes.go

```diff
@@ -0,0 +1,221 @@
+package testlib
```

Comment:
Instead of a package `testlib`, the pattern we're using is a package `test` under the directory where it's applicable. What about the package `pkg/operator/ceph/test`?